docs/Using_Datasets.md

```diff
@@ -118,7 +117,25 @@ dataset is linked to a publication listed in PubMed.*
 -------------
 
 
-Step 3: Data Scopes
+
+Step 3: Using Dataset favorites
+---------------
+
+Since R2 is hosting hundreds of datasets it could be handy to store in a quick way the datasets you often use. Clicking on the starred+ symbol will add the dataset you currently selected to a subset of favorite datasets.
```

Comment:
to store in a quick way the datasets you often use

to store the datasets you often use, in a preselection that is easily accessible